packages/config-loader/src/lib/secrets.ts

```diff
@@ -134,6 +140,35 @@ export async function readSecret(
 
     return String(value);
   }
+  if ('include' in secret) {
+    const [filePath, dataPath] = secret.include.split(/#(.*)/);
```

Comment:
Build says that `secret.include` is never and it's failing 🤷 

Wonder if we need to add `include` to the `secret` type? not sure where that comes from/.